pvm.py

```diff
@@ -1,18 +1,26 @@
 from datetime import datetime
 import os
+from pathlib import Path
 from pythonosc import dispatcher, osc_server
 import argparse
 from omxplayer.player import OMXPlayer
 import logging
 from logging.handlers import TimedRotatingFileHandler
 import sys
 
+# Place your videos in this folder for autostart
+HOME = str(Path.home()) + "/"  # The home directory, e.g. /home/pi/
+LOG_PATH = HOME + "PVM/log/"
+PEFIX_PATH = HOME + "Videos/"
```

Comment:
please fix this typo here and wherever else it appears
```suggestion
PREFIX_PATH = HOME + "Videos/"
```